lib/sqlite_ecto/error.ex

```diff
@@ -8,5 +8,15 @@ defmodule Sqlite.Ecto.Error do
   def exception({type, msg}), do: %__MODULE__{sqlite: {type, to_string(msg)}}
   def exception(msg), do: %__MODULE__{sqlite: msg}
 
+  require Logger
+
+  def to_constraints(%__MODULE__{sqlite: {:constraint, "UNIQUE constraint failed: " <> _}}) do
```

Comment:
I guess I am a little confused by when `to_constraints/1` is being called.  If I add the warning to the exception, users will still see the message about being unable to support constraints whether they are using them or not.  Is there some way to know that `unique_constraint/3` or `foreign_key_constraint/3` has been called?